test/compressionworker.ts

```diff
@@ -32,10 +32,12 @@ describe("Compression Worker Tests", () => {
 
   it("validates that events are batched for upload correctly when total length is below the limit", (done: DoneFn) => {
     let worker = createTestWorker();
-    let firstMockEventName = "FirstMockEvent";
-    let secondMockEventName = "SecondMockEvent";
-    let firstMockEvent = getMockEvent(firstMockEventName);
-    let secondMockEvent = getMockEvent(secondMockEventName);
+    let firstMockEventType = -2;
+    let secondMockEventType = -3;
```

Comment:
What are thiese "mock" event type here for and why do you need one for first and second?